app/common/controllers/login-controller.js

```diff
@@ -11,10 +11,10 @@ function (
     Authentication,
     $location
 ) {
```

Comment:
If you clean out this controller and just leave:
```
$scope.$emit('event:login:show:loginModal');
$location.url('/');
```
I **think** that should open the modal and render the main map page behind it. Try it?